include/alpaka/math/sqrt/Traits.hpp

```diff
@@ -27,7 +28,16 @@ namespace alpaka
             //#############################################################################
             //! The sqrt trait.
             template<typename T, typename TArg, typename TSfinae = void>
-            struct Sqrt;
+            struct Sqrt
+            {
+                ALPAKA_FN_HOST_ACC auto operator()(T const& ctx, TArg const& arg)
+                {
+                    alpaka::ignore_unused(ctx);
+                    // This is an ADL call. If you get a compile error here then your type is not suppored by the
```

Comment:
Typo: suppored instead of supported

Reply:
Are you bored? :D One comment would have been enough!